### ISSUE TITLE
MD037 Flags asterisks in HTML attributes

### ISSUE THREAD

**Issue by mtlynch** [DavidAnson/markdownlint#540]
First, thanks for all your excellent work on markdownlint! I ran into a minor bug today that I wanted to share.

If there are two asterisks in an HTML attribute, markdownlint will flag it as MD037/no-space-in-emphasis. This seems to be an error as there isn't markdown within HTML attributes:

### Example input

```text
<img src="cron.png" alt="cron job with a '0 0 3 * *' schedule">
```

### markdown-lint output

```bash
$ davidanson/markdownlint-cli2 example.md
markdownlint-cli2 v0.5.0 (markdownlint v0.26.1)
Finding: example.md
Linting: 1 file(s)
Summary: 2 error(s)
example.md:1:49 MD037/no-space-in-emphasis Spaces inside emphasis markers [Context: "* *"]
example.md:1 MD041/first-line-heading/first-line-h1 First line in a file should be a top-level heading [Context: "<img src="cron.png" alt="cron ..."]
```

**Comment by DavidAnson** [comment]
Thanks for the clear reproduction! There are a number of rules that use heuristics instead of the parser and so don't really understand the context of what they are reporting. I already have a helper that excludes violations like this from code spans; I should be able to do something similar for HTML.

**Comment by DavidAnson** [comment]
Oh, and thanks for the sponsorship!!
